scripts/run-rules.sh

```diff
@@ -22,7 +22,7 @@ OUTPUTDIR=$KB_DATA_DIR/$RULETYPE/
 RULES_RES_DIR=edu/ucdenver/ccp/kabob/build/$RULETYPE/
 
 echo Running rules
-mkdir --parents $OUTPUTDIR
+mkdir -p $OUTPUTDIR
```

Comment:
Sorry about this.  Did you encounter this on MacOS/BSD? 
Another reason to move to a proper build tool that abstracts away these kinds of platform inconsistencies.